### PR TITLE
The best ever version

### PR DESCRIPTION
**Pull request by ghost**
- [DEMO LINK](https://junglq.github.io/layout_catalog/)
- [TEST REPORT LINK](https://junglq.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,252 @@
...
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 1024px;
+  margin: 0 auto;
+  padding: 50px 40px;
+  // .cards__item
+  &__item {
+    margin-bottom: 46px;
+    margin-right: 48px;
```
Comment (AlinaMatuschak) [inline src/styles/main.scss:126]:
```suggestion
    margin: 0 48px 46px 0;
```

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,252 @@
...
+    align-items: center;
+    background-color: #00acdc;
+    border-radius: 5px;
+    border: 1px solid #00acdc;
+    color: #fff;
+    font-weight: 700;
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+    transition: all 0.3s ease 0s;
```
Comment (AlinaMatuschak) [inline src/styles/main.scss:187]: don't use `all` like on the previous task

File: src/index.html [hunk 3]
```diff
@@ -4,9 +4,284 @@
...
+          <div class="cards__item card">
+            <img class="card__img" src="images/imac.jpeg"
+              alt="imac">
+
+            <h2 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+
+            <div class="card__code">Код товара: 195434</div>
+
+            <div class="card__rating rating">
+              <div class="rating__stars">
```
Comment (AlinaMatuschak) [inline src/index.html:206]: add a class with modifier (--number) and keep it operational

**Review by AlinaMatuschak — CHANGES_REQUESTED** [review]
Good start 👌. Fix 8, 10 points on [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,252 @@
...
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+}
+
+.page {
+  flex: 1 1 auto;
+}
```
[the remark this answers is not in the document]
Comment (junglq) [reply src/styles/main.scss:54]: это для прибития футера

File: src/index.html [hunk 5]
```diff
@@ -2,11 +2,298 @@
...
+          <div class="cards__item card" data-qa="card">
+            <img class="card__img" src="images/imac.jpeg"
+              alt="imac">
+
+            <h2 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+
+            <div class="card__code">Код товара: 195434</div>
+
+            <div class="card__rating rating">
+              <div class="rating__stars rating__stars--1">
```
Comment (AlinaMatuschak) [inline src/index.html:72]: Look at the mockup of how many stars must be and fir rating__stars--1 class

Comment (junglq) [reply src/index.html:72]: я спецально сделал чтобы была видна разница

Comment (AlinaMatuschak) [reply src/index.html:72]: You can leave this. It's not so big deal. But fix 8, 10 points on [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)

**Review by aionova — APPROVED** [review]
you didn't include all the files with styles to the pull request

File: src/styles/blocks/cards.scss [hunk 6]
```diff
@@ -0,0 +1,78 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 1024px;
+  margin: 0 auto;
+  padding: 50px 40px;
+  // .cards__item
```
Comment (aionova) [inline src/styles/blocks/cards.scss:7]: no need in these comments.

File: src/styles/blocks/cards.scss [hunk 7]
```diff
@@ -0,0 +1,78 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 1024px;
+  margin: 0 auto;
```
Comment (aionova) [inline src/styles/blocks/cards.scss:5]: don't give margins and max-width to BEM-block. Give them to element class (page__cards)

File: src/styles/blocks/cards.scss [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+    text-decoration: none;
+    transition: color 0.3s ease 0s;
+    padding: 5px;
+    line-height: 16px;
+    &:hover {
+      background-color: transparent;
+      color: $blue;
+    }
+  }
+}
```
Comment (aionova) [inline src/styles/blocks/cards.scss:78]: add empty line

File: src/styles/blocks/header.scss [hunk 9]
```diff
@@ -0,0 +1,60 @@
+.header {
+  box-shadow: 0 2 4 rgba(0, 0, 0, 0.05);
+  background-color: #fff;
```
Comment (aionova) [inline src/styles/blocks/header.scss:3]: use variables for colors

File: src/styles/blocks/header.scss [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+
+  // .header__logo-link
+  &__logo-link {
+    display: block;
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
```
Comment (aionova) [inline src/styles/blocks/header.scss:17]: separate file for each bem-block

File: src/styles/blocks/header.scss [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+    content: "";
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: 4px;
+    background-color: $blue;
+    border-radius: 8px;
+  }
+}
```
Comment (aionova) [inline src/styles/blocks/header.scss:60]: empty line

File: src/styles/blocks/rating.scss [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+    }
+  }
+
+  // .rating__review
+  &__review {
+    font-size: $fontSmall;
+    line-height: 14px;
+    color: $darkBlue;
+  }
+}
```
Comment (aionova) [inline src/styles/blocks/rating.scss:36]: empty line

**Review by aionova — CHANGES_REQUESTED** [review]

File: src/styles/reset.scss [hunk 13]
```diff
@@ -0,0 +1,39 @@
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
```
Comment (aionova) [inline src/styles/reset.scss:4]: use * selector only for box-sizing

File: src/styles/blocks/cards.scss [hunk 14]
```diff
@@ -0,0 +1,69 @@
+.cards {
```
Comment (aionova) [inline src/styles/blocks/cards.scss:1]: cards is a separate BEM-block, so it should be located in a separate file

**Pull request closed without merging**